diff --git a/configure b/configure
index 9adfb97468..a62bee561a 100755
--- a/configure
+++ b/configure
@@ -122,8 +122,8 @@ create_environ() {
 : ${SBINDIR:="${PREFIX}/sbin"}
 : ${ETCDIR:="${SPREFIX}/etc"}
 : ${LIBDIR:="${SPREFIX}/lib"}
-: ${PKGCFG_LIBDIR:="\${exec_prefix}/lib"}
-: ${PKGCFG_INCDIR:="\${prefix}/include"}
+: ${PKGCFG_LIBDIR:='${exec_prefix}/lib'}
+: ${PKGCFG_INCDIR:='${prefix}/include'}
 : ${LIBEXECDIR:="${SPREFIX}/libexec"}
 : ${INCLUDEDIR:="${SPREFIX}/include"}
 : ${DATADIR:="${SPREFIX}/share"}
